dbms/src/Server/BgStorageInit.cpp

```diff
@@ -52,7 +51,7 @@ void BgStorageInitHolder::start(Context & global_context, const LoggerPtr & log,
         std::atomic<int> init_cnt = 0;
```

Comment:
The changes in this file LGTM.

Maybe we can make `do_init_stores` to be a function instead of a lambda function? It become too complicated for a lambda function now...

Reply:
Done